extension/options.html

```diff
@@ -1,23 +1,36 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Options</title>
-  <style>
-    body: { padding: 10px; }
-  </style>
+	<title>Options</title>
```

Comment:
Seems that these changes haven't been reverted. I'll try tomorrow. Sorry for the inconvenient.